documentation/release_notes.rst

```diff
@@ -8,6 +8,54 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.3
+=============
+
+New Features
+------------
+-  Added the Range-based versions of the following algorithms: ``all_of``, ``any_of``, ``count``, ``count_if``, ``equal``, ``move``, ``remove``, ``remove_if``, ``replace``, ``replace_if``.
+-  Added the following utility ranges (viewes): ``generate``, ``fill``, ``rotate``.
+
+Changes to Existing Features
+-----------------------------
+-  Improved performance of the ``discard_block_engine`` (including ``ranlux24``, ``ranlux48``, ``ranlux24_vec``, ``ranlux48_vec`` predefined engines) and ``normal_distribution`` RNG functionality.
+- Added two constructors to ``transform_iterator``: ``transform_iterator()`` and ``transform_iterator(iter)``.
+- ``transform_iterator`` meets the requirements of forward iterators instead of random-access iterator requirements.
+
+Fixed Issues
+------------
+-  Fixed parallel execution of ``swap_ranges`` algorithm with following C++ standard execution policies: ``unseq``, ``par``.
+-  Fixed the issue of scan-based algorithms, which caused memory corruption and double memory free operation on CPU devices with -O0 and -g options.
+-  Fixed incorrect behavior in the ``exclusive_scan`` algorithm that occurred when the input and ouput iterator ranges overlapped.
+-  Fixed error propagation for async runtime exceptions by consistently calling ``sycl::event::wait_and_throw`` internally.
+-  Fixed ``-Wreturn-std-move`` warning.
```

Comment:
This is not the warning but a compiler option that enables it. I would instead describe the warnings somehow.

Reply:
As I understand, this issue was reported by a person who downloaded oneDPL samples. The purpose of this wording is to explicitly notify them that the issue is fixed. @capatober, @dparanic what to you think about warning description?

Reply:
Let's see how it impacts users. Previously, they face it when enable some compiler options to see more warnings. We fixed one of them.

So, it can be:
> Fixed the warning: ``local variable will be copied despite being returned by name [-Wreturn-std-move]``

or
> Fixed the warning about copying local variable despite being returned by name.

I think the 1st one is better since one can easily find an information about that warning.

Reply:
I am fine with Andrey's 1st suggestion.